docker-compose.yaml

```diff
@@ -2,7 +2,7 @@ version: "3"
 
 services:
     zeebe:
-        image: camunda/zeebe:${CAMUNDA_CLOUD_VERSION:-1.1.3}
+        image: camunda/zeebe:${CAMUNDA_CLOUD_VERSION:-1.2.0}
```

Comment:
:x: We should also update the versions of Operate and Tasklist. Or, is the version updated by someone else?

Reply:
I guess that will be done by the release managers of those teams. But, you are right. Probably it doesn't make sense to update the zeebe version alone. We should wait until operate and task list are released.